.docker/ci/Dockerfile

```diff
@@ -69,8 +70,8 @@ RUN apt-get -qq update && \
 # Download moveit source and fetch all necessary dependencies
 RUN mkdir src && \
     cd $ROS_WS/src && \
-    # TODO(mlautman): This should be changed to use vcs in the future
-    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit2/master/moveit.rosinstall && \
+    wget -q https://raw.githubusercontent.com/henningkayser/moveit2/pr-moveit_core/moveit2.repos && \
```

Comment:
Yes, this should trigger an automated build (I already pushed the same one manually). Actually, I think i'll just push the .repos file here so that the build succeeds right away.